dist/images/kubectl-ko

```diff
@@ -949,159 +941,107 @@ env-check(){
+      echo "*********************nft filter ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param 2>/dev/null || :
 
-      echo "****************** legacy nat ************************"
-      kubectl exec $pod -n kube-system -- iptables -S -t nat || :
-      echo "********************* nft nat ************************"
-      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S -t nat || :
+      echo "****************** legacy nat ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-legacy -S -t nat >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
+      echo "********************* nft nat ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S -t nat >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param 2>/dev/null || :
 
     elif [[ "$sub_component_param" == "route" ]]; then
-      kubectl exec $pod -n kube-system -- ip route show || :
+      kubectl exec $pod -n kube-system -- ip route show > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "link" ]]; then
-      kubectl exec $pod -n kube-system -- ip -d link show || :
+      kubectl exec $pod -n kube-system -- ip -d link show > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "neigh" ]]; then
-      kubectl exec $pod -n kube-system -- ip n || :
+      kubectl exec $pod -n kube-system -- ip n > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "memory" ]]; then
-      kubectl exec $pod -n kube-system -- free -m || :
+      kubectl exec $pod -n kube-system -- free -m > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "top" ]]; then
-      kubectl exec $pod -n kube-system -- top -b -n 1 | head -n 20 | tail -n 10 || :
+      kubectl exec $pod -n kube-system -- top -b -n 1 > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "sysctl" ]]; then
-      kubectl exec $pod -n kube-system -- sysctl -a || :
+      kubectl exec $pod -n kube-system -- sysctl -a > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "netstat" ]]; then
-      kubectl exec $pod -n kube-system -- netstat -tunlp || :
+      kubectl exec $pod -n kube-system -- netstat -tunlp > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
```

Comment:
Append a file extension such as `.txt` so that the files can be opened directly in Windows/MacOS.

Reply:
add suffix with .log